src/components/KymaModules/KymaModulesCreate.js

```diff
@@ -349,6 +394,59 @@ export default function KymaModulesCreate({ resource, ...props }) {
         </MessageBox>,
         document.body,
       )}
+      {createPortal(
+        <MessageBox
+          type="Warning"
+          open={showManagedBox.isOpen}
+          onClose={() => {
+            setShowManagedBox({ isOpen: false, hide: false, onSave: false });
+          }}
+          titleText={showManagedBox?.onSave ? 'SIRIUS WARNING' : 'WARNING'}
+          actions={[
+            <Button
+              accessibleName={
+                showManagedBox?.onSave ? 'change-managed' : 'ok-managed'
+              }
+              design="Emphasized"
+              key={showManagedBox?.onSave ? 'change-managed' : 'ok-managed'}
+              onClick={() =>
+                showManagedBox?.onSave
+                  ? isEdited
+                    ? 'OK'
+                    : handleCreate()
+                  : setShowManagedBox({
+                      isOpen: false,
+                      onSave: false,
+                    })
```

Comment:
Nested ternaries are hard to read, I'd move it into a callback and separate the conditionals clearly. Also the `hide` prop is missing from `setShowManagedBox`.